front/src/filters/filters.js

```diff
@@ -0,0 +1,8 @@
+import Vue from 'vue';
+
+Vue.filter('getImgSrc', function (imgPath, baseUrl) {
+  return baseUrl + '/' + imgPath;
+});
+Vue.filter('spaceBetweenElement', function (titles) {
+  return titles.map(element => ' ' + element.firstName + ' ' + element.lastName).join();
+});
```

Comment:
it is unclear at all. spaceBetweenElement - this name doesn't tell about goal. it is about nothing
i am not sure that i understand correctly the goal but might be it is fullName and listStringify filters
so, it could be
```
Vue.filter('fullName', (source) => {
  return source.isArray(source) && source.length 
             ? source.map(({firstName, lastName}) => `${firstName} ${lastName}`)
             : `${source.firstName} ${source.lastName}`;
});
Vue.filter('listStringify', (sourceList) => {
  return sourceList.join(', ');
});
```
and then
`celebrities | fullName | listStringify`